server/statistics/hot_cache.go

```diff
@@ -77,15 +77,23 @@ func NewHotStoresStats() *HotStoresStats {
 // CheckRegionFlow checks the flow information of region.
 func (f *HotStoresStats) CheckRegionFlow(region *core.RegionInfo, kind FlowKind) []HotSpotPeerStatGenerator {
 	var (
-		generators   []HotSpotPeerStatGenerator
-		getBytesFlow func() uint64
-		getKeysFlow  func() uint64
-		bytesPerSec  uint64
-		keysPerSec   uint64
+		generators     []HotSpotPeerStatGenerator
+		getBytesFlow   func() uint64
+		getKeysFlow    func() uint64
+		bytesPerSec    uint64
+		keysPerSec     uint64
+		reportInterval uint64
 
 		isExpiredInStore func(region *core.RegionInfo, storeID uint64) bool
 	)
 
+	reportInterval = region.GetInterval().GetEndTimestamp() - region.GetInterval().GetStartTimestamp()
+
+	// ignores this region flow information if the report time interval is too short or too long.
+	if reportInterval < minHotRegionReportInterval || reportInterval > 3*RegionHeartBeatReportInterval {
```

Comment:
I'm not sure if this is necessary. /cc @nolouch

Reply:
It's necessary, but we should consider `isExpiredInStore(region)` in here.